File: examples-sdk/java/039-basic-observer-dashboard/Main.java
// 039-basic-observer-dashboard — Java SDK equivalent
// Compile: vil compile --from java --input 039-basic-observer-dashboard/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("observer-demo", 8080);
        ServiceProcess demo = new ServiceProcess("demo");
        demo.endpoint("GET", "/hello", "hello");
        demo.endpoint("POST", "/echo", "echo");
        server.service(demo);
        server.compile();
    }
}

File: examples-sdk/kotlin/039-basic-observer-dashboard/main.kt
// 039-basic-observer-dashboard — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 039-basic-observer-dashboard/main.kt --release

fun main() {
    val server = VilServer("observer-demo", 8080)
    val demo = ServiceProcess("demo")
    demo.endpoint("GET", "/hello", "hello")
    demo.endpoint("POST", "/echo", "echo")
    server.service(demo)
    server.compile()
}
